actionlib_lift_params: #sets params for elevator_server.cpp #default values
    cargo:
        intake_position: 0.04
        cargo_ship_position: 0.7
        rocket1_position: 0.03
        rocket2_position: 0.8
        rocket3_position: 1.5
    hatch:
        intake_position: 0.0
        intake_end_position: 0.0
        rocket1_position: 0.0
        rocket2_position: 0.77
        rocket3_position: 1.5
    climber3: #Used for level three climbs
        deploy_position: 0.95
        climb_position: 0.0
        climb_low_position: 0.17 #0.22
        climb_raise_position: 0.79
        deploy_timeout: 5.0
        climb_timeout: 15.0
        climb_low_timeout: 10.0
        running_forward_timeout: 10.0    
    climber2: #Used for level two climbs
        deploy_position: 0.95
        climb_position: 0.27
        climb_low_position: 0.5 
        climb_raise_position: 0.79
        deploy_timeout: 5.0
        climb_timeout: 15.0
        climb_low_timeout: 10.0
        running_forward_timeout: 10.0   
    max_speed: -1.0
    timeout: 10
    elevator_position_deadzone: 0.05
    climber_engage_pos: 0.1 #elevator position right where climber clicks in to engage TODO: figure this out
    wait_at_top_for_engage: 0.2
